CODE_OF_CONDUCT.adoc

```diff
@@ -0,0 +1,47 @@
+# Code of Conduct
+
+The uProtocol project follows the https://raw.githubusercontent.com/eclipse/.github/master/CODE_OF_CONDUCT.md[Eclipse Code of Conduct] and https://www.eclipse.org/projects/handbook/[Eclipse Foundation Handbook]. These two resources should be read before continuing. .
+
+Below we define addition code of conduct and responsibilities of committers and project leads. 
+
+NOTE: Any violations to this code of conduct should be reported to the project lead and/or the Eclipse Foundation.
+
+
+## Committers
+https://www.eclipse.org/projects/handbook/#roles-cm[Eclipse-uProtocol Committer] holds plays a vital role to ensure ensure contributions from others (and themselves) follow the vision and mission of the project, and adhere to this code of conduct. 
+In the next section we will elaborate on their expected area of expertise and responsibilities, along with how committers are nominated and retired.
+
+[#area-of-expertise]
+### Committer's Area of Expertise
+Committers are chosen based on merit and the area of expertise that they can steward (maintain). List below is not exhaustive and shall update as the project evolves:
+
+1. **Specifications** (up-spec, up-core-api): Ensuring the vision/mission of the project is upheld, process is followed, and the specifications are maintained and updated. Concerns are discussed and addressed w.r.t backwards compatibility support and new features.
+2. **Programming language:** Experts in a particular language who help to review and comment on contributions for a given supported language (Java, Kotlin, Rust, C++, Python, etc...)
+3. **Implementation:** Experts in a particular implementation of uProtocol (ex. Android, Zenoh, Sommr/SOME/ip, Cloud, etc...)
+
+NOTE: Committers are strongly encouraged to contribute in more than one area defined above.
+
+
+### Responsibilities
+Additional R&R of a uProtocol committers are as follows:
```

Comment:
Bit of jargon, common jargon, but all the same maybe could be: "roles and responsibilities (R&R)"